launcher/InstanceImportTask.cpp

```diff
@@ -262,6 +262,25 @@ void InstanceImportTask::extractFinished()
     }
 }
 
+bool installIcon(QString root, QString instIcon)
+{
+    auto importIconPath = IconUtils::findBestIconIn(root, instIcon);
+    if (importIconPath.isNull() || !QFile::exists(importIconPath))
+        importIconPath = IconUtils::findBestIconIn(root, "icon.png");
```

Comment:
Modrinth/CurseForge packs can't add anything outside of .minecraft so I'm not sure what this is for

Reply:
prism export
I didn't want to rewrte that logic for all of them